components/event-bus/internal/knative/util/kn-eventing.go

```diff
@@ -336,12 +336,12 @@ func resendMessage(httpClient *http.Client, channel *messagingV1Alpha1.Channel,
 	return nil
 }
 
-func makeChannel(name string, namespace string, labels *map[string]string) *messagingV1Alpha1.Channel {
+func makeChannel(prefix string, namespace string, labels *map[string]string) *messagingV1Alpha1.Channel {
```

Comment:
maps are pointers, so `*map` is a big no-no.
Redundant `string`.
```suggestion
func makeChannel(prefix, namespace string, labels map[string]string) *messagingV1Alpha1.Channel {
```